tests/pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+markers =
+    online: marks test as requiring access to the internet (to run offline: '-m "not online"'
```

Comment:
Missing closing `)`.
The end message is somewhat misleading. It will not "run offline", but skip online tests entirely.